app-modules/engagement/resources/views/components/message-center-inbox.blade.php

```diff
@@ -82,8 +83,8 @@ class="col-span-full h-full overflow-y-auto rounded-bl-lg rounded-br-lg rounded-
                                         </div>
                                         <div class="w-full text-xs font-normal text-gray-700 dark:text-gray-400">
                                             Last engaged at
-
-                                            {{ Carbon::parse($educatable->latest_activity)->format('g:ia - M j, Y') }}
+                                            @php $timezone =  app(DisplaySettings::class)->getTimezone() @endphp
+                                            {{ Carbon::parse($educatable->latest_activity)->setTimezone($timezone)->format('g:ia - M j, Y') }}
```

Comment:
```suggestion
                                            {{ Carbon::parse($educatable->latest_activity)->setTimezone(app(DisplaySettings::class)->getTimezone())->format('g:ia - M j, Y') }}
```
I am curious if this works. Cleans the lines up a bit IMO. What do you think?

Reply:
@Orrison Yes I had this thing in my mind, but I am kind of person who does not like to pass long paths inside variable, rather taking it into a variable and then using it makes it look more readable to me. But if you want I can obviously make the changes.

Reply:
@jayushi-canyon no it's fine. Your reasoning is sound.